src/main/java/fitr/command/EditEntryCommand.java

```diff
@@ -138,8 +140,8 @@ private void editExercise(ExerciseList exerciseList, String arguments) {
 
```

Comment:
Can check if its a letter here so the next line wont throw NumberFormatException